tests/cpp.test

```diff
@@ -793,3 +793,4 @@
 60045  align_asterisk_after_type_cast.cfg   cpp/align_asterisk_after_type_cast.cpp
 60046  align_continuation_left_shift.cfg    cpp/align_continuation_left_shift.cpp
 60047  align_default_after_override.cfg     cpp/align_default_after_override.cpp
+60048  semicolon-removal-after-ternary-operator.cfg cpp/semicolon-removal-after-ternary-operator.cpp
```

Comment:
Missing newline at EOF. Also, please use the existing `mod_remove_extra_semicolon-t.cfg` :slightly_smiling_face:.

Reply:
done